docs/zh/deployment/deploy_manually.md

```diff
@@ -176,9 +182,73 @@ displayed_sidebar: docs
       default_replication_num = 1
       ```
 
+## 第二步：（存算分离）启动 CN 服务
+
+:::note
+
+只能将 CN 节点添加到存算分离集群中。不建议在存算一体集群中添加 CN 节点，否则可能导致未知行为。
+
+:::
+
+Compute Node（CN）是一种无状态的计算服务，本身不存储数据。您可以通过添加 CN 节点为查询提供额外的计算资源。您可以使用 BE 部署文件部署 CN 节点。CN 节点自 v2.4 版本起支持。
```

Comment:
CN for shared-data is supported since V3.0 ?

Reply:
CN node itself was released in 2.4. It was added to shared-nothing before shared-data was released in 3.0.